src/main/kotlin/com/nextstep/blackjack/domain/Player.kt

```diff
@@ -0,0 +1,43 @@
+package com.nextstep.blackjack.domain
+
+import com.nextstep.blackjack.domain.Card.Companion.BLACKJACK_ACE_CORRECTION
+import com.nextstep.blackjack.domain.Card.Companion.BLACKJACK_ACE_JUDGE_THRESHOLD
+import com.nextstep.blackjack.domain.Card.Companion.BLACKJACK_NUMBER
+
+data class Player(val name: String) {
+    private var _cards: List<Card> = mutableListOf()
+
+    fun draw(drawCards: List<Card>) {
+        _cards = _cards.plus(drawCards)
+    }
+
+    fun isBust(): Boolean {
+        return calculateScore() > BLACKJACK_NUMBER
+    }
+
+    fun getCards(): List<Card> {
+        return _cards.toList()
+    }
```

Comment:
toList() 방어적 복사 👍 

참고로 저는 이런 종류의 함수는 [backing 프로퍼티](https://kotlinlang.org/docs/properties.html#backing-properties)로 만들어 가변 타입은 감추고 불변 타입은 외부에 노출하도록 하는 것을 선호합니다. 읽기/쓰기 연산이나 유효성 검사 등의 로직이 포함되어 있는 경우에는 함수로, 단순 클래스 내부 값을 노출하는 경우에는 프로퍼티의 getter로 만드려고 해요.
```suggestion
        val cards: List<Card> get() = _cards.toList()
```

Reply:
오 이거 제대로 한건지 한번 확인부탁드릴게요